chia/cmds/wallet.py

```diff
@@ -97,11 +99,11 @@ def send_cmd(
     default=None,
 )
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
-def show_cmd(wallet_rpc_port: int, fingerprint: int) -> None:
+def show_cmd(wallet_rpc_port: Optional[int], farmer_rpc_port: int, fingerprint: int) -> None:
```

Comment:
Why are you adding `farmer_rpc_port` here?

Reply:
Not sure, i think that is a mistake. I'll revert